contracts/build.rs

```diff
@@ -33,6 +33,24 @@ fn main() {
                 )),
             )
     });
+    generate_contract_with_config("BalancerV2WeightedPoolFactory", |builder| {
+        builder
+            .with_contract_mod_override(Some("balancer_v2_factory"))
```

Comment:
naming-nit: I would maybe specify that this is specifically a weighted pool factory (as we may eventually add support for other pools like stable pools, which would mean we we need a `balancer_v2_stable_pool_factory`).

```suggestion
            .with_contract_mod_override(Some("balancer_v2_weighted_pool_factory"))
```

Reply:
I can do this, although it would have been the plan to change it when necessary.

Reply:
Maybe `balancer_v2_pool_factory` at least, since I find just `factory` a little vague (is it for the `Vault`? For BAL token? etc.).